device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttIotHubConnectionTest.java

```diff
@@ -46,7 +46,7 @@
     final String deviceKey = "test-devicekey?&test";
     final String apiVersionPrefix = "?api-version=";
     final String API_VERSION = apiVersionPrefix + TransportUtils.IOTHUB_API_VERSION;
-    final String PNP_API_VERSION = apiVersionPrefix + TransportUtils.IOTHUB_API_VERSION_PREVIEW + "&digital-twin-model-id=" + modelId;
```

Comment:
Why are you switching from a preview API version to normal API version? The preview branch should have a preview API version